vulnfeeds/cmd/alpine/Dockerfile

```diff
@@ -25,7 +25,7 @@ COPY ./ /src/
 RUN go build -o alpine-osv ./cmd/alpine/
 
 
-FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine
+FROM gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine@@sha256:d5da0344b23d03a6f2728657732c7a60300a91acaad9b8076c6fd30b1dfe1ff4
```

Comment:
Is double `@` the correct syntax? I thought it was just one `@`

Reply:
🤦‍♂️ thanks for catching that, fixed in [5d1c751](https://github.com/google/osv.dev/pull/2484/commits/5d1c75185134d7f0e95e729e3494b32872e8b9ea)